gitops/triggers/triggers-crds/interceptors/services.yaml

```diff
@@ -0,0 +1,422 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
same as for other crds

Reply:
Same reply :wink: 
Closing this one as resolved.